jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/api/client/clientrequestcontext/JAXRSClientIT.java

```diff
@@ -882,7 +882,7 @@ protected void checkFilterContext(ClientRequestContext context) throws Fault {
           //Shouldn't find first value when separator character is incorrect
           assertFalse(context.containsHeaderString("header1", ";", "value1"::equalsIgnoreCase));
           //Test regular expression
-          assertFalse(context.containsHeaderString("header1", "; | ,", "value2"::equalsIgnoreCase));
+          assertFalse(context.containsHeaderString("header1", ";|,", "value2"::equalsIgnoreCase));
```

Comment:
`"value,value1 , value2,Value3,white space".split(";|,") = ["value", "value1 ", " value2", "Value3", "white space "]`
Why is this supposed to assert false?

Reply:
It shouldn't.  That was not my intent.  thanks